### PR TITLE
Initial Config

### PR DESCRIPTION
**Pull request by MR1R0B0T**
Create Pull Request

### CODE REVIEW

File: Calculator.MR1R0B0T/CalculatorApp/CalculatorApp/Program.cs [hunk 1]
```diff
@@ -4,7 +4,116 @@ internal class Program
...
+                    MultiplicationGame();
+                    break;
+                case "4":
+                    Console.Clear();
+                    DivisionGame();
+                    break;
+                default:
+                    Console.Clear();
+                    Console.WriteLine("Out of bounds. Please try again.");
+                    Environment.Exit(0);
```
Comment (edvaudin) [inline Calculator.MR1R0B0T/CalculatorApp/CalculatorApp/Program.cs:54]: Typing in a wrong input should not close the program, it should re-prompt the user for a valid input.

File: Calculator.MR1R0B0T/CalculatorApp/CalculatorApp/Program.cs [hunk 2]
```diff
@@ -4,7 +4,116 @@ internal class Program
...
+
+        //Addition
+        public static void AdditionGame()
+        {
+            Console.WriteLine("\t\t\t\t\t\tWelcome to the Addition Game\n\n\n");
+            Console.WriteLine("Please input the 2 numbers: ");
+            Console.Write("First Number: ");
+            int num1 = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Second Number: ");
+            int num2 = Convert.ToInt32(Console.ReadLine());
```
Comment (edvaudin) [inline Calculator.MR1R0B0T/CalculatorApp/CalculatorApp/Program.cs:67]: This does not handle validating where the user has entered two numbers that can be parsed. Currently, there is an unhandled exception if the user types in an invalid input. You should consider creating a separate method that is responsible for validating that the user's input is indeed a number. If they do not enter a number, the program should re-prompt them for a valid input.

File: Calculator.MR1R0B0T/CalculatorApp/CalculatorApp/Program.cs [hunk 3]
```diff
@@ -4,7 +4,116 @@ internal class Program
...
+        //Division
+        public static void DivisionGame()
+        {
+            Console.WriteLine("\t\t\t\t\t\tWelcome to the Division Game\n\n\n");
+            Console.WriteLine("Please input the 2 numbers: ");
+            Console.Write("First Number: ");
+            int num1 = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Second Number: ");
+            int num2 = Convert.ToInt32(Console.ReadLine());
+            int quotient = num1 / num2;
```
Comment (edvaudin) [inline Calculator.MR1R0B0T/CalculatorApp/CalculatorApp/Program.cs:113]: This currently does not protect from the user entering 0 as the second number, which results in an unhandled exception. The program should ensure that either the user cannot enter 0 for the second number, or it should display NaN or something similar.

**Review by edvaudin — CHANGES_REQUESTED** [review]
Congratulations on building a Calculator! The main theme of my comments will be focusing on validating user input. Currently, the user can crash the program in a few ways such as by entering non-numbers or by dividing by zero. The program should catch these and prompt the user to enter a valid input. 

Fixing the above will provide an opportunity to look at extracting some more methods. If you look closely at each of the operator methods, you will notice that they have several steps in common, such as prompting the user for input, collecting the user's input and then performing a calculation. These processes can be extracted into common methods. If you need any help with this, feel free to reach out to me (Xarvus on Discord).

There are some opportunities to reduce the number of Console.WriteLine() calls, however this is more minor.

I hope that all helps! Good luck!

**Comment by TheCSharpAcademy** [comment]
@MR1R0B0T I'm closing this for admin purposes, feel free to reopen if you want us to review.

**Pull request closed without merging**